### ISSUE TITLE
Metrics to extract

### ISSUE THREAD

**Issue by KaustubhG** [SE17GroupH/GitHub_analysis#4]
Currently we only have issues name

What we need:
1. Issues and comment metrics, Labels (maybe) : working on these
2. Milestone related metrics (overdue issue) : done
3. Commit metrics (size, time, frequency) : done

I'm working on extracting the metrics.

**Comment by thegreyd** [comment]
@DevArenaCN I've uploaded the issue csv files here https://github.com/SE17GroupH/GitHub_analysis/blob/master/SE17GroupH_Zap_issues.csv Try opening it and Tell me if you need anything else.

**Comment by thegreyd** [comment]
Also I was directly committing to master branch, now I've created my own branch

**Comment by DevArenaCN** [comment]
@thegreyd I think that's everything I needed. I've created scripts for bar charts using that data, let me know if you need any more graphs. I'll finish it tonight.

**Comment by thegreyd** [comment]
cool. can i see how it looks? or how do i run it on my computer?

**Comment by DevArenaCN** [comment]
@thegreyd Yeah, they are in the graph folder, all the scripts for generating the graphs are in the R folder, you just need to download R on your computer, and copy paste my script into the R console, and you got yourself a graph

**Comment by thegreyd** [comment]
I also added the pull request field which indicates if an issue is a pull rquest. That can be used to differentiate between issue and pull request

**Comment by DevArenaCN** [comment]
@thegreyd I thought there was an online site that can generate that graph right? We can use that one instead of having to writing script for the plot

**Comment by KaustubhG** [comment]
Graphs to generate for issue data @DevArenaCN 
Bar charts:
1. open issues over time
2. closed issues over time

(number of issues) over (issue duration time) : 
3.Issues generated less than 24 hours and 
4. Issues generated more than 24 hours

5. (range of comments) over (number issues):
for example there were 10 issues with 10-20 comments.
This is exactly like the duration graph but with comments.



Pie charts:
1. Issues without labels, issues with bug labels, issues with other labels
2. percent of issues created by each user
3. percent of issues assigned to each user
4. percent of issues with milestones, percent of issues without milestones
